back/app/policies/initiative_policy.rb

```diff
@@ -15,12 +15,12 @@ def resolve
   end
 
   def index_xlsx?
-    active? && admin?
+    active? && UserRoleService.new.can_moderate_initiatives?(user)
```

Comment:
Why is this different from using `can_moderate?` that we've used elsewhere?

Reply:
In this case not, because there is no initiative.